app/src/main/java/org/mifos/mobilebanking/ui/fragments/RegistrationFragment.java

```diff
@@ -153,10 +153,7 @@ private boolean areFieldsValidated() {
                                         < etPassword.getText().toString().length()) {
             Toaster.show(rootView,
                     getString(R.string.error_validation_cannot_contain_leading_or_trailing_spaces,
-                    getString(R.string.password)));
-            return false;
-        } else if (etUsername.getText().toString().trim().length() < 6) {
```

Comment:
@therajanmaurya Fix 1: There we 2 different checks for Username Length (one with length 5, other with length 6)